docs/MetricExportApi.md

```diff
@@ -437,7 +445,7 @@ Name | Type | Description  | Notes
 
 ### HTTP request headers
 
-- **Content-Type**: application/json
+- **Content-Type**: Not defined
```

Comment:
I wonder what would cause this?

Reply:
I saw this.  I'm assuming I broke it somehow trying to add the content-type to each call.  I'll look into it.

Reply:
I investigated this.  "Not defined" is apparently a normal value here for requests that don't have a body such as get requests.  The request in question here is a POST request but it doesn't have a body.   I did an upgrade on the library that detects these and I'm assuming there was a change such that POST requests without a body are detected.

The line here seems pretty awkward to me.  I would prefer if it just left it out if there was not specified content-type.  That's a change we could probably make to the template in a separate PR.